extension-manifest-v2/src/modules/renew.js

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
please move it to already existing folder. Otherwise packaging script will require an update.

Reply:
I am not sure what you mean here. The `modules/` path has been already used before: https://github.com/ghostery/ghostery-extension/blob/main/extension-manifest-v2/src/modules/autoconsent.js

The file is bundled, so the `modules/` is not in the result code.